### ISSUE TITLE
[REQ] Floating Window Zoom - upon re-start the app should remember the prior zoom level so users don't have to re-adjust to their preference every time

### ISSUE THREAD

**Issue by aaciii** [cagnulein/qdomyos-zwift#1112]

**Comment by cagnulein** [comment]
i thought the zoom could be removed. do you think it could be useful?

**Comment by aaciii** [comment]
I found it useful personally to make everything slightly smaller rather than keep making the floating window larger... but that could be done differently.  

However, I think the zoom should not just reduce the text size - but reduce the actual box size as well. The whole floating window, text and background.   Since some users may prefer a small display.

My goal is to fit everything with only a small black border around everything.  Ideally this will be automatic, so the user doesn't have to fuss with dimensions and zoom and scroll, etc.

**Comment by cagnulein** [comment]
now it's the zoom of the web view itself.

yes automatic resize of the window is one of the thing that must be implemented before put this in production i guess

also other users asked me about target tiles, peloton offset, start pause buttons...so they want a qz floating interface at all :)

**Comment by aaciii** [comment]
Well... I would suggest that if people want all the tons of data then they have the existing interface ;).  

If the floating window becomes immense, it defeats the purpose of being able to see key stats on top of other apps.

But I suppose you could take the approach of whatever tiles are selected in QZ also appear as a row on the floating view.

**Comment by cagnulein** [comment]
customization is the key of qz :) 
we have to find a easy way to make everyone happy :) 
using a qz floating has sense for the users that wants auto resistance

**Comment by timkarnold** [comment]
What about just having a font size setting in the UI Options? Removing Zoom would also rectify an issue I am experiencing that the Close button is hidden/obstructed by the zoom controls.

<img width="395" alt="Screen Shot 2023-01-03 at 12 06 29 PM" src="https://user-images.githubusercontent.com/1531526/210405591-dcb8cd25-5b0f-40cf-8fc8-216c69cfdbe5.png">

**Comment by cagnulein** [comment]
yes i vote for font size too against the zoom. so I have to play with CSS

Guys could you tell me a which size are you using for the actual floating one? Because it is dependent from the DPI i would like to understand which DPI has your tablet

**Comment by cagnulein** [comment]
I did this for the qz ai companion. Did you try it @aaciii ?
